plugins/item_licenses/server/__init__.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
+#  Copyright Kitware Inc.
+#
+#  Licensed under the Apache License, Version 2.0 ( the "License" );
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+###############################################################################
+
+import six
+
+from girder import events
+from girder.constants import AccessType, SettingDefault
+from girder.models.model_base import ValidationException
+from girder.utility.model_importer import ModelImporter
+
+from .constants import PluginSettings, PluginSettingsDefaults
+from .rest import getLicenses
+
+
+def validateString(value):
+    """
+    Make sure a value is a string.
+
+    :param value: the value to coerce into a string if it isn't already.
+    :returns: the string version of the value.
+    """
+    if value is None:
+        value = ''
+    if not isinstance(value, six.string_types):
+        value = str(value)
```

Comment:
Just make this check for and coerce to `six.text_type`. It should always be `str` in Python3, and making it always `unicode` in Python2 should be harmless.